README.md

```diff
@@ -7,11 +7,37 @@ Docker is a tool for creating a container, in this case running the ROS noetic i
 To use Docker, make sure you have downloaded [Docker Engine](https://docs.docker.com/engine/install/ubuntu/) and [Docker compose](https://docs.docker.com/compose/install/).
 To build the Docker image, navigate to this folder (Vortex-ASV) int the terminal and execute the command 
 '''
-sudo docker-compose build
+$ sudo docker-compose build
 '''
+Note that you only need to build the image once unless changes have been made to the Dockerfile since the last time you built.
 After building the image, you can run any service using
 '''
-sudo docker-compose up -d <service_name>
+$ sudo docker-compose up -d <service_name>
 '''
+Running without a service name runs every container in the docker-compose file.
```

Comment:
Not sure if this is entirely correct. Docker documentation sucks

Reply:
Yes and no. The compose file lists services which will be made into images when building. Not sure if there is a way of spinning up all images that is registered.